contracts/deploy/sign.py

```diff
@@ -59,7 +66,8 @@ def format_int(value, size):
             msg += arg
         elif isinstance(arg, str):
             if is_0x_prefixed(arg):
-                msg += decode_hex(arg)
+                arg = pad_left(remove_0x_prefix(arg), 64, '0')
```

Comment:
I don't think that's correct, as there are `byte8`'s etc. in Solidity which are not padded. I would just leave it like it is (the caller can pad if he needs to).